### DISCUSSION TITLE
what mean "destination_storage" and "message_age", doc not find !

### DISCUSSION THREAD

**Discussion by startjava** [apache/pulsar#19722] [Q&A] [answered]
in BacklogQuota.BacklogQuotaType

**Comment by nodece** [comment]
@momo-jun Could you check that?

**Comment by momo-jun** [comment, marked as answer]
@startjava you can find answers from the search results of the reference site, like [here](https://pulsar.apache.org/reference/#/next/pulsar-admin/namespaces?id=set-backlog-quota).
